dev_scripts_helpers/github/dockerized_sync_gh_repo_settings.py

```diff
@@ -59,9 +107,15 @@ def get_repository_settings(
         Get the current settings of the repository `repo`.
 
         :param repo: GitHub repository object
-        :return: dictionary containing repository settings
+        :return: dictionary containing current settings of the
+            repository E.g., { "name": str, "default_branch": str,
```

Comment:
The linter probably messed this up.

Can we make a reference to the data structure REPO_SETTING_KEYS in the comment for DRY?